### ISSUE TITLE
rrwebRecord and plain Javascript CDN script

### ISSUE THREAD

**Issue by LorenzoRogai** [getsentry/sentry-rrweb#32]
Hi,

rrweb also provides the recorder JS and it would be good to only include this script instead of the whole recorder/replay bundle

`<script src="https://cdn.jsdelivr.net/npm/rrweb@latest/dist/record/rrweb-record.min.js"></script>`

By changing the rrweb.record function call to rrwebRecord it will work

Could you please also distribute a .js file without require/module export statements? I am using plain Javascript and it would be good to just include both script like this

```
<script src="https://cdn.jsdelivr.net/npm/rrweb@latest/dist/record/rrweb-record.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/@sentry/rrweb"></script>
```

Right now for sentry/rrweb i'm using a modified version hosted locally

Thank you

**Comment by Yuyz0112** [comment]
Is there some require/module export statements in this file? https://cdn.jsdelivr.net/npm/rrweb@latest/dist/record/rrweb-record.min.js

**Comment by LorenzoRogai** [comment]
No, that is fine. The require statements are inside the sentry/rrweb dist file

**Comment by onetom** [comment]
instead of vendoring a modified version of the rrweb file, i would just prepare the expected environment, by stubbing out the `require` function and the `module` var, then "redirecting" `rrweb` to `rrwebRecord`:
```html
<script>function require() {}; module = {}; rrweb = {record(...args) {rrwebRecord(...args)}}</script>
<script src="https://cdn.jsdelivr.net/npm/@sentry/rrweb"></script>
```
or something along those lines; I haven't tried it yet.

**Comment by davidflypei** [comment]
So messing around with it. I was able to get it to function without a modified version by working with @onetom's idea.

This worked:
```
<script src="https://cdn.jsdelivr.net/npm/rrweb@latest/dist/record/rrweb-record.min.js" crossorigin="anonymous"></script>
<script>
  function require(thing) {
    if (thing == 'rrweb') {
      return {record(...args) {rrwebRecord(...args)}};
    }
  };
  let module = {};
</script>
<script src="https://cdn.jsdelivr.net/npm/@sentry/rrweb" crossorigin="anonymous"></script>
```
If I did it @onetom's way I would get a rrweb is undefined error so I had to make `require()` return `rrweb.record()` as @onetom put it "redirected" to `rrwebRecord()`. BAM!!! It's alive!

it was sort of working when I had a custom scuffed version that I made from the typescript. I could not get even a basic integration to work by doing a basic integration. Something like this...
```
class SentryRRWeb {
  constructor() {
    this.name = 'SentryRRWeb';
  }

  setupOnce() {
    Sentry.addGlobalEventProcessor(event => {
      const self = Sentry.getCurrentHub().getIntegration(SentryRRWeb);
      // Run the integration ONLY when it was installed on the current Hub
      if (self) {
        if (event.message === 'Happy Message2') {
          event.message = `\\o/ ${event.message} \\o/`;
        }
      } else {
        console.error('SentryRRWeb not self');
      }
      return event;
    });
  }
}
```
Would not I guess install? `if (self)` would always be false for some reason. But if I did something stupid at the start like starting it like:
```
class SentryRRWeb {
  id = 'SentryRRWeb';
  name = SentryRRWeb.id;
```
It would somehow work because it was installed with the name undefined... But would throw errors by itself because of `name = SentryRRWeb.id;` for obvious reasons.

TL;DR
If you want to use this in a browser without a custom build then do:
```
<script src="https://cdn.jsdelivr.net/npm/rrweb@latest/dist/record/rrweb-record.min.js" crossorigin="anonymous"></script>
<script>
  function require(thing) {
    if (thing == 'rrweb') {
      return {record(...args) {rrwebRecord(...args)}};
    }
  };
  let module = {};
</script>
<script src="https://cdn.jsdelivr.net/npm/@sentry/rrweb" crossorigin="anonymous"></script>
```

**Comment by bruno-garcia** [comment]
We're releasing the Sentry Replay integration via the CDN.
Please note this repository has been superseded and will no longer be updated.

Please check https://github.com/getsentry/sentry-javascript/tree/master/packages/replay#sentry-session-replay instead, we're happy respond to issues, review and merge PRs there.
